src/auth-service/controllers/create-user.js

```diff
@@ -286,6 +287,73 @@ const createUser = {
       });
     }
   },
+  deleteMobileUserData: async (req, res) => {
+    try {
+      let responseFromDeleteAppData;
+      logText("We are deleting the app data.....");
+      const { uid } = req.query;
+      const { ct } = req.query;
+      
+      if (!uid) {
```

Comment:
Hi @BenjaminSsempala these input validations can be done from the respective `routes/*` files

Reply:
Thanks @Baalmart,
I'd actually included them there too, I've removed these ones now to reduce the repetition

Reply:
Alright, thanks @BenjaminSsempala